java/ql/lib/semmle/code/java/frameworks/apache/Collections.qll

```diff
@@ -182,6 +204,695 @@ private class ApacheKeyValueModel extends SummaryModelCsv {
+          ".queue;TransformedQueue;true;transformingQueue;;;Element of Argument[0];Element of ReturnValue;value",
+          ".queue;UnmodifiableQueue;true;unmodifiableQueue;;;Element of Argument[0];Element of ReturnValue;value"
+        ]
+  }
+}
+
+/**
+ * Value-propagating models for the package `org.apache.commons.collections4.set`.
+ */
+private class ApacheSetModel extends SummaryModelCsv {
+  override predicate row(string row) {
+    row =
+      ["org.apache.commons.collections4", "org.apache.commons.collections"] +
+        [
+          // Note that when lambdas are supported we should have more models for TransformedNavigableSet
+          ".set;AbstractNavigableSetDecorator;true;AbstractNavigableSetDecorator;;;Element of Argument[0];Element of Argument[-1];value",
+          ".set;AbstractSetDecorator;true;AbstractSetDecorator;;;Element of Argument[0];Element of Argument[-1];value",
+          ".set;AbstractSortedSetDecorator;true;AbstractSortedSetDecorator;;;Element of Argument[0];Element of Argument[-1];value",
+          ".set;CompositeSet$SetMutator;true;add;;;Argument[2];Element of Argument[0];value",
+          ".set;CompositeSet$SetMutator;true;add;;;Argument[2];Element of Element of Argument[1];value",
+          ".set;CompositeSet$SetMutator;true;addAll;;;Element of Argument[2];Element of Argument[0];value",
+          ".set;CompositeSet$SetMutator;true;addAll;;;Element of Argument[2];Element of Element of Argument[1];value",
+          ".set;CompositeSet;true;CompositeSet;(Set);;Element of Argument[0];Element of Argument[-1];value",
+          ".set;CompositeSet;true;CompositeSet;(Set[]);;Element of Element of Argument[0];Element of Argument[-1];value",
+          ".set;CompositeSet;true;addComposited;(Set);;Element of Argument[0];Element of Argument[-1];value",
+          ".set;CompositeSet;true;addComposited;(Set,Set);;Element of Argument[0];Element of Argument[-1];value",
+          ".set;CompositeSet;true;addComposited;(Set,Set);;Element of Argument[1];Element of Argument[-1];value",
+          ".set;CompositeSet;true;addComposited;(Set[]);;Element of Element of Argument[0];Element of Argument[-1];value",
+          ".set;CompositeSet;true;toSet;;;Element of Argument[-1];Element of ReturnValue;value",
+          ".set;CompositeSet;true;getSets;;;Element of Argument[-1];Element of Element of ReturnValue;value",
+          ".set;ListOrderedSet;true;listOrderedSet;(Set);;Element of Argument[0];Element of ReturnValue;value",
+          ".set;ListOrderedSet;true;listOrderedSet;(List);;Element of Argument[0];Element of ReturnValue;value",
+          ".set;ListOrderedSet;true;asList;;;Element of Argument[-1];Element of ReturnValue;value",
+          ".set;ListOrderedSet;true;get;;;Element of Argument[-1];ReturnValue;value",
+          ".set;ListOrderedSet;true;add;;;Argument[1];Element of Argument[-1];value",
+          ".set;ListOrderedSet;true;add;;;Element of Argument[1];Element of Argument[-1];value",
```

Comment:
There's also an addAll op not specified on Collection

Reply:
It seems this line was meant to be `addAll`